operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -384,6 +387,55 @@ function checkDirNotExistsOrEmpty() {
   trace "Exiting checkDirNotExistsOrEmpty"
 }
 
+# limit the file extensions in the model directories
+
+function checkModelDirectoryExtensions() {
+  trace "Entering checkModelDirectoryExtensions"
+
+  cd ${IMG_MODELS_HOME}
+  counter=$(ls  -I  *.yaml -I *.zip -I *.properties | wc -l)
+  if [ $counter -ne 0 ] ; then
+        trace SEVERE "Directory ${IMG_MODELS_HOME} contains files with unsupported extension. Extension must be" \
+          ".yaml, .properties, or .zip"
+        trace SEVERE $(ls -I  *.yaml -I *.zip -I *.properties)
+        exitOrLoop
+  fi
+  if [ -d ${WDT_CONFIGMAP_ROOT} ] ; then
+    cd ${WDT_CONFIGMAP_ROOT}
+    counter=$(ls  -I  *.yaml -I *.properties | wc -l)
+    if [ $counter -ne 0 ] ; then
+          trace SEVERE "Directory ${WDT_CONFIGMAP_ROOT} contains files with unsupported extension. Extension must be" \
+            ".yaml or .properties"
+          trace SEVERE $(ls -I  *.yaml -I *.properties)
+          exitOrLoop
+    fi
+  fi
+
+  trace "Exiting checkModelDirectoryExtensions"
+}
+
+# Check for WDT version
+
+function checkWDTVersion() {
+  trace "Entering checkWDTVersion"
+  unzip -c ${WDT_ROOT}/lib/weblogic-deploy-core.jar META-INF/MANIFEST.MF > /tmp/wdtversion.txt || exitOrLoop
+  local wdt_version="$(grep "Implementation-Version" /tmp/wdtversion.txt | cut -f2 -d' ' | tr -d '\r' )" || exitOrLoop
+  if  [ ! -z ${wdt_version} ]; then
+    versionGE ${wdt_version} ${WDT_MINIMUM_VERSION}
+    if [ $? != "0" ] ; then
+      trace SEVERE "Model in Image domain requires Weblogic Deploy Tool minimum version ${WDT_MINIMUM_VERSION}. " \
+      "The version in the image is ${wdt_version}, you need to update the Weblogic Deploy Tool version in the image."
+      exitOrLoop
+    fi
+  else
+    trace SEVERE "Unknown Weblogic Deploy Tool version. Model in Image domain requires Weblogic Deploy Tool " \
+      "minimum version ${WDT_MINIMUM_VERSION}"
+    exitOrLoop
+  fi
```

Comment:
Just in case the version format changes, or there's some other bug, please provide an environment variable to bypass the check, and also print an informative message if the bypass is in effect.    

And add to each trace above.  ". To bypass this check, set environment variable 'WDT_BYPASS_WDT_VERSION_CHECK' to 'true'."

Reply:
done

Reply:
I don't see the text I requested in the SEVERE.

Reply:
done

Reply:
thanks